internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -303,11 +303,11 @@ func newBool(b bool) *bool {
 	return bp
 }
 
-const cmdTemplate = `/bin/mkdir -p {{ dirname .DBPath }} && \
+const cmdTemplate = `mkdir -p {{ dirname .DBPath }} && \
 {{- range $i, $item := .BundleItems }}
-/bin/opm registry add -d {{ $.DBPath }} -b {{ $item.ImageTag }} --mode={{ $item.AddMode }}{{ if $.CASecretName }} --ca-file=/certs/cert.pem{{ end }} --skip-tls={{ $.SkipTLS }} && \
+opm registry add -d {{ $.DBPath }} -b {{ $item.ImageTag }} --mode={{ $item.AddMode }}{{ if $.CASecretName }} --ca-file=/certs/cert.pem{{ end }} --skip-tls={{ $.SkipTLS }} && \
 {{- end }}
-/bin/opm registry serve -d {{ .DBPath }} -p {{ .GRPCPort }}
```

Comment:
Will no longer the bin be installed by default in the bin dir scaffold in the project?